### ISSUE TITLE
Tracing outcome of Research cruise (campaigns)  

### ISSUE THREAD

**Issue by Ketilkj** [datacite/freya#62]
As a funding agency, I would like to trace the outcome of my financial contribution to a marine research cruise (Cruise ID) by tracking the data generated (data-PID) and articles (publication-PID) published, as well as physical samples taken (IGSN) and the repository (organization ID), where these samples are physically stored. In this regard, I would also like to track the future data and publications generated from these samples.

**Comment by markusstocker** [comment]
Could include instruments used on the cruise.

**Comment by Ketilkj** [comment]
true, but I think "instruments" would fit better with another user-story. I don't think funders would consider "Instruments-deployed" as a valued outcome of a cruise in itself. 
The value of adding instruments is more for the organizations and researchers that can track the deployment of their instrument in time and space.

**Comment by mfenner** [comment]
This issue was moved to https://www.pidforum.org/t/tracing-outcome-of-research-cruise-campaigns/73
